cutobjects/ITS3/CutsDstoKKpi_ITS3.C

```diff
@@ -12,13 +12,160 @@
+    anacutsval[0][0]=0.25;
+    anacutsval[1][0]=0.4;
+    anacutsval[2][0]=0.4;
+    anacutsval[3][0]=0.;
+    anacutsval[4][0]=0.;
+    anacutsval[5][0]=0.;
+    anacutsval[6][0]=0.02;
+    anacutsval[7][0]=0.02;
+    anacutsval[8][0]=0.;
+    anacutsval[9][0]=0.97;
+    anacutsval[10][0]=0.;
+    anacutsval[11][0]=100000.0;
+    anacutsval[12][0]=0.015;
+    anacutsval[13][0]=0.001;
+    anacutsval[14][0]=0.;
+    anacutsval[15][0]=1.;
+    anacutsval[16][0]=0.;
+    anacutsval[17][0]=0.;
+    anacutsval[18][0]=4.;
+    anacutsval[19][0]=0.9;
+
+    anacutsval[0][1]=0.25;
+    anacutsval[1][1]=0.4;
+    anacutsval[2][1]=0.4;
+    anacutsval[3][1]=0.;
+    anacutsval[4][1]=0.;
+    anacutsval[5][1]=0.;
+    anacutsval[6][1]=0.02;
+    anacutsval[7][1]=0.02;
+    anacutsval[8][1]=0.;
+    anacutsval[9][1]=0.95;
+    anacutsval[10][1]=0.;
+    anacutsval[11][1]=100000.0;
+    anacutsval[12][1]=0.015;
+    anacutsval[13][1]=0.001;
+    anacutsval[14][1]=0.;
+    anacutsval[15][1]=1.;
+    anacutsval[16][1]=0.;
+    anacutsval[17][1]=0.;
+    anacutsval[18][1]=3.;
+    anacutsval[19][1]=0.9;
+
+    anacutsval[0][2]=0.25;
+    anacutsval[1][2]=0.4;
+    anacutsval[2][2]=0.4;
+    anacutsval[3][2]=0.;
+    anacutsval[4][2]=0.;
+    anacutsval[5][2]=0.;
+    anacutsval[6][2]=0.02;
+    anacutsval[7][2]=0.02;
+    anacutsval[8][2]=0.;
+    anacutsval[9][2]=0.;
+    anacutsval[10][2]=0.;
+    anacutsval[11][2]=100000.0;
+    anacutsval[12][2]=0.015;
+    anacutsval[13][2]=0.001;
+    anacutsval[14][2]=0.;
+    anacutsval[15][2]=1.;
+    anacutsval[16][2]=0.;
+    anacutsval[17][2]=0.;
+    anacutsval[18][2]=0.;
+    anacutsval[19][2]=0.9;
+ 
+  
+    AliRDHFCutsDstoKKpi* analysiscuts=new AliRDHFCutsDstoKKpi();
+    analysiscuts->SetName("AnalysisCuts");
+    analysiscuts->SetTitle("Cuts for Ds Analysis and CF");
+    analysiscuts->SetUsePreSelect(1);
+    analysiscuts->SetPtBins(nptbins+1,ptbins);
+    analysiscuts->SetCuts(nvars,nptbins,anacutsval);
+    analysiscuts->AddTrackCuts(esdTrackCuts);
+
+    analysiscuts->SetUseCentrality(AliRDHFCuts::kCentOff); //
+    analysiscuts->SetTriggerClass("");//dont use for ppMB/ppMB_MC
+    analysiscuts->ResetMaskAndEnableMBTrigger();//dont use for ppMB/ppMB_MC
+    analysiscuts->SetTriggerMask(AliVEvent::kAny);
+
+    analysiscuts->SetUsePID(true);
+    analysiscuts->SetPidOption(0); //0=kConservative,1=kStrong
+
+    analysiscuts->SetOptPileup(false);
+    analysiscuts->SetMinPtCandidate(ptmin);
+    analysiscuts->SetMaxPtCandidate(ptmax);
+    analysiscuts->SetRemoveDaughtersFromPrim(false);
+
+    std::cout<<"This is the object I'm going to save:"<<nptbins<<std::endl;
+
+    analysiscuts->PrintAll();
+
+    TFile* fout=new TFile(Form("DstoKKpiCutsITS2_filttree_kAny_consPID_pt%0.f_%0.f.root",ptmin,ptmax),"recreate");
+    fout->cd();
+    analysiscuts->Write();
+    fout->Close();
+
+    return analysiscuts;
+}
+
+AliRDHFCutsDstoKKpi* MakeFileForCutsDs_FiltTree_ITS2_Data(double ptmin=1., double ptmax=50.) {

```

Comment:
Very minor comment about the name: it would be good to have data with ITS2, but unfortunately we only have fo ITS1 :wink: